src/pip/_internal/operations/install/legacy.py

```diff
@@ -73,7 +73,16 @@ def install(
             )
 
         if not os.path.exists(record_filename):
-            logger.debug('Record file %s not found', record_filename)
+            logger.warning('Record file %s not found', record_filename)
```

Comment:
We should also mention the name of the package.

Reply:
The record filename *should* include that. :P

You're right -- explicitly mentioning the pacakge name will more directly surface relevant info to the user. :)

Reply:
Unless I'm missing something it's just in a temporary directory with a generic name.